app/javascript/components/disable-offline/component.spec.js

```diff
@@ -0,0 +1,26 @@
+import { mountedComponent, screen } from "test-utils";
+import { fromJS } from "immutable";
+
+import DisableOffline from "./component";
+
+describe("<DisableOffline /> - Component", () => {
+  const props = {
+    overrideCondition: true,
+    text: "offline"
+  };
+
+  it("renders element in offline state if offline", () => {
+    mountedComponent(
+      <DisableOffline {...props}>
+        <div>element</div>
+      </DisableOffline>,
+      fromJS({
+        connectivity: {
+          online: false,
+          serverOnline: true
+        }
+      })
+    );
+    expect(screen.getByText("element")).toBeInTheDocument();
```

Comment:
You should check if a tooltip is rendered. "element will always be there". Also missing the test for if the application is online "renders element in online state if online"